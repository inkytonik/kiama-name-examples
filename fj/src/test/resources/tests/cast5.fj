class A extends Object {
  A() {
    super();
  }
}
(A) new A().m()    // type of unknown expression is unknown
