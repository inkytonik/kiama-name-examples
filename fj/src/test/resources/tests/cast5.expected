src/test/resources/tests/cast5.fj:6:1:info: (A)new A() ...: A
(A) new A().m()    // type of unknown expression is unknown
^
src/test/resources/tests/cast5.fj:6:1:info: (A)new A() ...: unknown
(A) new A().m()    // type of unknown expression is unknown
^
src/test/resources/tests/cast5.fj:6:5:info: new A(): A
(A) new A().m()    // type of unknown expression is unknown
    ^
unknown
